packages/dds/shared-object-base/src/sharedObjectKernel.ts

```diff
@@ -289,6 +293,10 @@ export interface KernelArgs {
 	 * {@inheritdoc @fluidframework/container-definitions#IDeltaManager.lastSequenceNumber}
 	 */
 	readonly lastSequenceNumber: () => number;
+	/**
+	 * The minVersionForCollab specified when instantiating the ContainerRuntime.
```

Comment:
This comment (and the same in other places) isn't very descriptive. I believe there are comments in the compatibility file somewhere that explains what this is and how it should be used by consumers. You could {@link} that here. Additionally, some info on how shared objects should use it will be helpful.